### PR TITLE
Re-dockerised ras-collection-instrument service

### PR DESCRIPTION
**Pull request by madeye-matt**
Can someone take a careful look at the changes in Pipfile.lock - they came about when I did 'pipenv install xlsxwriter' but they don't look right to me ...

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -27,6 +27,7 @@ pika = "==0.10.0"
 cryptography = "==1.9"
 flask-httpauth = "==3.2.3"
 six = "==1.10.0"
+xlsxwriter = "*"
```
Comment (andrewmil) [inline Pipfile]: We should probably pin this to a specific version

File: config/config-docker.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+service:
```
Comment (benjefferies) [inline config/config-docker.yaml]: Do we need the config for docker or will overriding the environmental variables be good enough?

Comment (JamesGardiner) [reply config/config-docker.yaml]: Unfortunately this service is not 12FA and doesn't get configured via it's environment. Instead a YAML file is used to configure the service as per https://github.com/ONSdigital/ras-collection-instrument/blob/20cce0c57516e373adf4d3b67321f40e696a717a/app.py#L14

Comment (JamesGardiner) [reply config/config-docker.yaml]: I will raise a tech debt issue here and on trello to get this app configured via it's environment.

Comment (benjefferies) [reply config/config-docker.yaml]: I think it has some support around env variables but don't know how it works fully https://github.com/ONSdigital/ras-common-utils/blob/master/ras_common_utils/ras_config/ras_config.py#L38

File: Pipfile (hunk 1 above)
Comment (LJBabbage) [reply Pipfile]: It should be 1.0.2

File: config/config-docker.yaml (hunk 2 above)
Comment (madeye-matt) [reply config/config-docker.yaml]: I was unable to configure this application from the environment so what I did was to change run.py to allow an env variable to specify the config file so you could override the config from the environment.  I ran this approach past a couple of people and they were happy it was an improvment on what went before :-) I can't see what code changes are required here (short of rewriting the whole configuration piece, which is beyond the scope of this card)

**Comment by benjefferies** [comment]
@madeye-matt pipfile.lock file looks out of date

**Pull request merged**